athena_cfn/athena-bucket-cfn.yml

```diff
@@ -0,0 +1,233 @@
+# Purpose:  Expose Metadata from a single Quilt bucket in Athena
```

Comment:
It would be nice to have (or backlog) a service role for this process. So users don't need all the permissions. Example https://docs.quiltdata.com/advanced/technical-reference#before-you-install-quilt

Reply:
Does this cover that? If not, can you add what else is missing? 

https://app.asana.com/0/1201334960547652/1200634565890676/f